linksight/accounts/templates/registration/activation_email_subject.txt

```diff
@@ -0,0 +1 @@
+Confirm your email
```

Comment:
Maybe prefix with [LinkSight]?

Reply:
I thought that but it wasn't uniform with the other subject: "Welcomet to LinkSight". And parang weird ung "[LinkSight] Welcome to LinkSight"?

Reply:
well it could just be "[LinkSight] Welcome!" haha